docs/data/advanced-components/overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Advanced Components - Overview
+---
+
+# Advanced Components - Overview
+
+<p class="description">A collection of fast and powerful components.</p>
+
+## Licenses
+
+While MUI Core is entirely licensed under MIT, MUI X serves a part of its components under a commercial license.
+Please pay attention to the license.
+
+### Community Plan
+
+Published under [MIT license](https://tldrlegal.com/license/mit-license) and [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf)
+
+The packages of the **Community** Plan are:
+
+- [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid)
+- [`@mui/x-date-pickers`](https://www.npmjs.com/package/@mui/x-date-pickers)
+
+### Pro Plan
+
+This plan contains the features that are at the limit of what the open-source model can sustain. From a price perspective, the plan is designed to be accessible to most professionals.
+
+See [Pricing](https://mui.com/store/items/material-ui-pro/) for details on purchasing licenses.
+
+The packages of the **Pro** Plan are:
+
+- [`@mui/x-data-grid-pro`](https://www.npmjs.com/package/@mui/x-data-grid-pro)
+- [`@mui/x-date-pickers-pro`](https://www.npmjs.com/package/@mui/x-date-pickers-pro)
+
+The features exclusives to the **Pro** Plan are marked with the <span class="plan-pro"></span> icon across our documentation.
+
+<div class="only-light-mode">
+  <img src="/static/x/commercial-header-icon-light.png" style="width: 579px; margin-bottom: 2rem;" alt="">
+</div>
+<div class="only-dark-mode">
+  <img src="/static/x/commercial-header-icon-dark.png" style="width: 560px; margin-bottom: 2rem;" alt="">
+</div>
+
+### 🚧 Premium Plan
+
+This plan contains the most advanced features.
+
+The features exclusives to the **Premium** Plan are marked with the <span class="plan-premium"></span> icon across our documentation.
+The **Premium** Plan also contains all the features of the **Pro** Plan
+
+## MIT vs. commercial
+
+_How do we decide if a feature is MIT or commercial?_
+
+We have been building MIT React components since 2014,
+and have learned much about the strengths and weaknesses of the MIT license model.
+The health of this model is improving every day. As the community grows, it increases the probability that developers contribute improvements to the project.
+However, we believe that we have reached the sustainability limits of what the model can support for advancing our mission forward. We have seen too many MIT licensed components moving slowly or getting abandoned. The community isn't contributing improvements as fast as the problems deserved to be solved.
+
+We are using a commercial license to forward the development of the most advanced features, where the MIT model can't sustain it.
+A solution to a problem should only be commercial if it has no MIT alternatives.
+
+For each set of component, you can check the feature comparison table:
+
+- [Data Grid](/x/react-data-grid/getting-started/#feature-comparison)
```

Comment:
Should we directly link to [the pricing page](https://mui.com/pricing/) which has the most advanced feature comparison UI ?

Right now we have 3 lists:

- https://mui.com/pricing/
- https://mui.com/components/data-grid/getting-started/#feature-comparison
- https://mui.com/components/data-grid/#features

I would at least drop the 3rd one (not in this PR)

Reply:
I do not know if we should remove mui.com/components/data-grid/#features

For sure we should not link to this section because it is less informative than feature comparison. But it seems to be a good way to give an overview of what is available in the component without having to navigate on a big table

Reply:
> I think the mui.com/pricing page is the best place to send people.

@samuelsycamore would you keep https://mui.com/components/data-grid/getting-started/#feature-comparison
And if so, for which usages ?

Reply:
@flaviendelangle is there any info on that page that isn't included on the Pricing page? If it's all there already then I think this #feature-comparison section is redundant. I would lean toward keeping all of this info on the Pricing page as the single source of truth.

Reply:
- https://mui.com/pricing/
- https://mui.com/components/data-grid/getting-started/#feature-comparison
- https://mui.com/components/data-grid/#features

The 1st two are almost identical (but the 2nd is DataGrid only)
Le 3rd one is a partial list 

I think we could have two at most
With :
- The pricing page with a very detailed collapsible list, that serve as source of truth for our community
- On section in each component page focusing only on the most important / most exciting features of the pro / premium, to serve as an entry point for Community users to give them a taste of what we have and give them a reason to go to the full list